src/components/CharacterForm/CharacterForm.js

```diff
@@ -131,6 +175,8 @@ const CharacterForm = ({ setOpen }) => {
               name="eyeColor"
               label="Eye Color"
               placeholder="Enter eye color"
+              rules={rules.eyeColor}
+              errors={errors}
```

Comment:
You shouldn't be passing the entire `errors` object to your input components. Components should only get the props they need